ludwig/utils/backward_compatibility.py

```diff
@@ -291,6 +366,12 @@ def upgrade_deprecated_fields(config: Dict[str, Any]):
     for feature in config.get(INPUT_FEATURES, []) + config.get(OUTPUT_FEATURES, []):
         _upgrade_feature(feature)
 
+    for feature in config.get(INPUT_FEATURES, []):
+        _upgrade_encoder_decoder_params(feature, True)
```

Comment:
nit: Use `_upgrade_encoder_decoder_params(feature, input_feature=True)` and `_upgrade_encoder_decoder_params(feature, input_feature=False)` so that readers don't have to look up `_upgrade_encoder_decoder_params` to figure out what that extra boolean is for.

Reply:
This is a very good callout, thanks Justin!